inst/private/python_header.py

```diff
@@ -23,6 +23,7 @@ def myerr(e):
 try:
     import sympy
     import sympy as sp
+    from sympy import __version__ as sv
```

Comment:
maybe `as spver`?  or `as sympyver`?

Then `Version(sympyver) >= `... is also more readable.